app/src/openxr/cpp/OpenXRLayers.cpp

```diff
@@ -242,4 +242,40 @@ OpenXRLayerEquirect::GetNextStructureInChain() const {
 }
 #endif
 
+// OpenXRLayerPassthrough;
+
+OpenXRLayerPassthroughPtr
+OpenXRLayerPassthrough::Create(const VRLayerPassthroughPtr& aLayer, XrPassthroughFB passthrough) {
+  auto result = std::make_shared<crow::OpenXRLayerPassthrough>();
+  result->vrLayer = aLayer;
+  result->passthroughInstance = passthrough;
+
+  return result;
+}
+
+void
+OpenXRLayerPassthrough::Init(JNIEnv *aEnv, XrSession session, vrb::RenderContextPtr &aContext) {
+  XrPassthroughLayerCreateInfoFB layerCreateInfo = {XR_TYPE_PASSTHROUGH_LAYER_CREATE_INFO_FB};
+  layerCreateInfo.passthrough = passthroughInstance;
+  layerCreateInfo.purpose = XR_PASSTHROUGH_LAYER_PURPOSE_RECONSTRUCTION_FB;
+  layerCreateInfo.flags = XR_PASSTHROUGH_IS_RUNNING_AT_CREATION_BIT_FB;
+  CHECK_XRCMD(OpenXRExtensions::sXrCreatePassthroughLayerFB(session, &layerCreateInfo, &xrLayer));
```

Comment:
Same comment about the dependency with the FB extension

Reply:
I think `OpenXRLayerPassthrough` could be thought of as an abstraction, able to accommodate similar extensions that use a layer for passthorugh. 

The problem is that at this point, since it is the only extension we know about and support, it is difficult to nail a potentian abstraction layer for passthrough that would work for all other devices.

So I propose we keep it simple and implement this extension, and when the next device comes along, we will have more info to make a good design for the abstraction.

WDYT?